__EXAMPLES/interfaces/ring_section.py

```diff
@@ -16,7 +16,7 @@
 import sys
 
 # BLonD Common import
-sys.path.append('./../../../')
+sys.path.append('./../../../')  # Add blond_common to PYTHONPATH if not done
```

Comment:
```suggestion
sys.path.append('../../../')  # Add blond_common to PYTHONPATH if not done
```

Personally I wouldn't do this at all.  This will only run in the correct directory, it will be an issue if someone copies it to another directory and tries to run it.

Reply:
Removed path manip in example